flexget/plugins/sites/torrentday.py

```diff
@@ -0,0 +1,184 @@
+    'tvDVDrip': 33,
+    'tvMOBILE': 46,
+    'tvPACKS': 14,
+    'tvSDx264': 26,
+    'tvHDx264': 7,
+    'tvX265': 34,
+    'tvXVID': 2
+}
+
+
+class UrlRewriteTorrentday(object):
+    """
+        Torrentday urlrewriter and search plugin.
+
+        torrentday:
+          uid: xxxxxxxxxxxxx  (required)    NOT YOUR LOGIN. find this in your browser's cookies
+          passkey: xxxxxxxxx  (required)    NOT YOUR PASSWORD. see previous
+          cfduid: xxxxxxxxxx  (required)    AGAIN IN THE COOKIES
+          rss_key: xxxxxxxxx  (required)    get this from your profile page
+          category: xxxxxxxx
+
+          Category can be one of 
+            ID from browsing site OR 'name'
+            movies:
+              mov480p, movHD, movBD, movDVD,
+              movMP4, movNonEnglish, movPACKS,
+              movSDx264, movX265, movXVID
+            tv:
+              tv480p, tvBRD, tvDVD, tvDVDrip,
+              tvMOBILE, tvPACKS, tvSDx264, 
+              tvHDx264, tvX265, tvXVID
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'rss_key': {'type': 'string'},
+            'uid': {'type': 'string'},
+            'passkey': {'type': 'string'},
+            'cfduid': {'type': 'string'},
+            'category': {
+                'oneOf': [
+                    {'type': 'integer'},
+                    {'type': 'string', 'enum': list(CATEGORIES)},
+                ]
+            },
+        },
+        'required': ['rss_key', 'uid', 'passkey', 'cfduid'],
+        'additionalProperties': False
+    }
+
+    # urlrewriter API
+    def url_rewritable(self, task, entry):
+        url = entry['url']
+        if url.find('.torrent'):
+            return False
+        if url.startswith('https://www.torrentday.com'):
+            return True
+        return False
+
+    # urlrewriter API
+    def url_rewrite(self, task, entry):
+        if 'url' not in entry:
+            log.error("Didn't actually get a URL...")
+        else:
+            log.debug("Got the URL: %s", entry['url'])
+        if entry['url'].startswith('https://www.torrentday.com/browse'):
+            # use search
+            results = self.search(task, entry)
+            if not results:
+                raise UrlRewritingError("No search results found")
+            entry['url'] = results[0]['url']
+
+    @plugin.internet(log)
+    def search(self, task, entry, config=None):
+        """
+        Search for name from torrentday.
+        """
+
+        if not isinstance(config, dict):
+            config = {}
+        categories = config.get('category', 'all')
+        # Make sure categories is a list
+        if not isinstance(categories, list):
+            categories = [categories]
+        # If there are any text categories, turn them into their id number
+        categories = [c if isinstance(c, int) else CATEGORIES[c] for c in categories]
+        filter_url = '&cata=yes&c%s=1&clear-new=1' % ','.join(str(c) for c in categories)
```

Comment:
Not mandatory, but I prefer passing URL params as a dict to requests, makes it more readable:
```python
params = { 'cata': 'yes', 'c%s' % ','.join(str(c) for c in categories): 1, 'clear-new': 1}
```
Then add it with `params=params` in the requests call. Just a suggestion